wallet/core/src/wallet/mod.rs

```diff
@@ -59,6 +67,7 @@ pub struct MultisigWalletFileV0<'a, T: AsRef<[u8]>> {
     pub encrypted_mnemonics: Vec<EncryptedMnemonic<T>>,
     pub xpublic_keys: Vec<&'a str>, // includes pub keys from encrypted
     pub required_signatures: u16,
+    #[allow(dead_code)]
```

Comment:
Why is this necessary here?

Reply:
Removed.

I accidentally left this there as a part of the effort to remove `pub` from various internal data structures, which in this case didn't work out as these data structures were used cross-crate in the workspace.  We need `pub (workspace)` :)